src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.file.FileReader;
+import core.basesyntax.file.FileWriter;
+import core.basesyntax.file.impl.FileReaderImpl;
+import core.basesyntax.file.impl.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.impl.BalanceOperation;
+import core.basesyntax.operation.impl.OperationStrategyImpl;
+import core.basesyntax.operation.impl.PurchaseOperation;
+import core.basesyntax.operation.impl.ReturnOperation;
+import core.basesyntax.operation.impl.SupplyOperation;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE_PATH = "finalReport.csv";
```

Comment:
```suggestion
    private static final String OUTPUT_FILE_PATH = "src/main/resources/finalReport.csv";
```
place it in the resources directory